### PR TITLE
[PR] Adjust breadcrumb display

### PR DESCRIPTION
**Pull request by jeremyfelt**
* Hide breadcrumbs completely on the home page.
* Remove the current item on all pages.

### CODE REVIEW

File: includes/breadcrumb.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Remove the current (last) item from the breadcrumb trail before output.
+ *
+ * Forked with thanks from https://github.com/mtekk/Breadcrumb-NavXT-Extensions
+ *
+ * @since 0.3.7
+ *
+ * @param \bcn_breadcrumb_trail $trail The object containing the breadcrumb trail.
+ */
+function remove_current_item( $trail ) {
```
Comment (philcable) [inline includes/breadcrumb.php:16]: I think we're missing the action hook for this function :)

**Review by philcable — CHANGES_REQUESTED** [review]
Looking good! I left one inline comment - and, we're currently displaying breadcrumbs in the footer, too, so the condition in `parts/footers.php` should probably be updated as well.

File: includes/breadcrumb.php (hunk 1 above)
Comment (jeremyfelt) [reply includes/breadcrumb.php:16]: 🤦‍♂️

**Comment by jeremyfelt** [comment]
Thanks for the catch(es)! @philcable #reviewmerge pt 2?

**Review by philcable — APPROVED** [review]
👍  Looks good!

**Pull request merged**